lms/envs/common.py

```diff
@@ -3938,3 +3938,9 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 
 ######################## Setting for content libraries ########################
 MAX_BLOCKS_PER_CONTENT_LIBRARY = 1000
+
+######################## Setting for django-countries ########################
+# django-countries provides an option to make the desired countries come up in
+# selection forms, if left empty countries will come up in ascending order as before
+# https://github.com/SmileyChris/django-countries#show-certain-countries-first
```

Comment:
Can you include an example in this comment? It's unclear from this or the linked doc exactly what goes into this list. I assume it's an array of two-digit country codes but that's not stated.

Reply:
sure, let me phrase this better and add example.

Reply:
@bradenmacdonald How is it looking now?

Reply:
Much better, thanks!